src/connection.ts

```diff
@@ -210,6 +211,34 @@ export class Connection extends JSForceConnection {
     return super._baseUrl();
   }
 
+  /**
+   * Will deploy a recently validated deploy request
+   *
+   * @param options.id = the deploy ID that's been validated already from a previous checkOnly deploy request
+   * @param options.rest = a boolean whether or not to use the REST API
+   */
+  public async deployRecentValidation(options: recentValidationOptions): Promise<JsonCollection> {
```

Comment:
I don't really know anything about this code base but it seems strange to me that this method would be inside `connection.ts` and the `Connection` class. Is this something all connections will need/use?

Reply:
It belongs in JSForce but we aren't going to put it there until we move over to v2. These will be removed in the v2 of core that uses v2 of jsforce. @WillieRuemmele can you put comments on these methods you are adding saying that, and that it is just to add the rest option for existing soap operations in jsforce.